app/jsx/components/AuthorListComp.jsx

```diff
@@ -13,16 +13,27 @@ class AuthorListComp extends React.Component {
     this.props.changeTab(tab_id)
   }
 
+  // Expects an array containing hashes with a 'name' attribute [{name: "Stone, Elizabeth C.", }]
+  // Returns array of just names, with first name prepended with a title
+  titledNames = (title, array) => {
+    return array.map((x, i) => {
+      return (i==0) ? `${title}(s): ` + x.name : x.name })
+  }
+
   render() {
     let p = this.props,
         year = p.pubdate.match(/\d{4}/),
-        authorlist = p.authors.map((author, i) => { return (<li key={i+author.name}>{author.name}</li>) })
+        t_authornames = (p.authors && !p.author_hide) ? this.titledNames("Author", p.authors) : [""],
+        t_editornames = p.editors ? this.titledNames("\xa0\xa0Editor", p.editors) : [""],
```

Comment:
Out of curiosity, why are non-breaking-spaces (`\xa0`) desirable here?

Reply:
I think the extra spacing at the beginning of each set of contributor types allows you to see the groupings better since it is all wrapped together in the same paragraph. Seemed like the best way to do this, or we could bold the headings of each section. I'll check w/Justin to see what he thinks.